.github/workflows/windows.yml

```diff
@@ -10,38 +10,44 @@ jobs:
     runs-on: windows-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Cache Nasm install
-      uses: actions/cache@v1
-      id: cache-nasm
-      env:
-        cache-name: cache-nasm
-      with:
-        path: "C:\\Program Files\\Nasm"
-        key: cache-nasm-v1
-        restore-keys: cache-nasm-v1
     - name: Cache LLVM install
-      uses: actions/cache@v1
+      uses: actions/cache@v2
```

Comment:
Didn't get the point of this change. Could you explain to me?

Reply:
1. Why _not_ using v2, since it is the updated? :D
2. I explained in Slack, but to clarify here too: v1 doesn't cache properly. And worse, when it caches, not everytime it sets `cache-hit=true`. That's why some things were executing everytime even if there was an `if steps.<something>.cache-hit == 'true'`.

Reply:
lol, okay, thanks